portfolio/src/main/webapp/index.html

```diff
@@ -87,14 +85,20 @@ <h1 id="introduction">My name is Gábor Fuisz</h1>
     </div>
     <div id="comment-section" class="full-page">
         <h1>Comments</h1>
-         <form action="/data" method="POST">
+         <form id="new-comment-form" action="/data" method="POST">
             <span>Your name: </span>
             <input type="text" name="name">
             <br>
             <span>Your comment: </span><br>
             <textarea  name="content"></textarea><br>
             <input type="submit" value="Comment"/>
          </form>
+         <a href="/login" id="toggle-login"></a>
     </div>
   </body>
+  <script src="jquery-3.5.1.min.js"></script>
+  <script src="script.js"></script>
+  <script src="fetch.js"></script>
+  <script src="comment_section.js"></script>
+  <script src="authentication.js"></script>
```

Comment:
make all of these absolute with a `/` in front.
What made you decide to _self-host_ and move them to the bottom?

Reply:
I've read it makes the page load quicker
The self-host certainly does.

Reply:
cool. I agree with moving them to the button making it load quicker. Self-hosting doesn't necessarily do that because you might already have the exact same javascript in your cache from other websites that use the same CDN. But it has other advantages. Certainly a good move.

Reply:
See e.g. https://stackoverflow.com/a/3832498

Reply:
Hmm interesting. I was aware of caching, but I did not know that there is a bottleneck on the queries to a single server in browsers. I think I'll do something like this: https://stackoverflow.com/a/3832530

Reply:
Actually you don't have to do this anymore. It's what I was meaning by some of it is outdated. Sorry for not being more specific.
HTTP 2.0 has a multiplexing feature that allows clients to basically download an unlimited amount of resources at the same time over a single connection to a server/domain. Therefore all those optimizations that many big websites did to split resources into several subdomains are not necessary anymore. (And HTTP 2 support is very good already across browsers. See https://caniuse.com/#feat=http2)

This stackoverflow answer covers this a bit:
https://stackoverflow.com/questions/36835972/is-the-per-host-connection-limit-raised-with-http-2#:~:text=With%20HTTP%2F2%2C%20browsers%20open,but%20it%20is%20virtually%20unlimited.